### PR TITLE
Adds the Scav-class Drifter

### PR DESCRIPTION
**Pull request by Sylphily**
## About The Pull Request
adds the scav class ship, designed for salvaging ruins, but made from long ago, and has worn bones.
it is a ISV faction
scavmk1

## Why It's Good For The Game
cool ship

## Changelog
:cl:
add: Scav-classed Drifter 
/:cl:

### CODE REVIEW

**Comment by Sylphily** [comment]
fuck forgot desc

**Comment by Sylphily** [comment]
any one knows why this is failing?

**Review by Fikou — CHANGES_REQUESTED** [review]
this needs to use the TGM map format

**Comment by Sylphily** [comment]
Yes I know the bridge has no wide band

**Comment by ZephyrTFA** [comment]
Try to avoid having thin diagonal walls or windows

**Comment by ZephyrTFA** [comment]
image

**Comment by Sylphily** [comment]
Literally can't without losing it's shape and keep room

**Comment by PiperDoots** [comment]
> Literally can't without losing it's shape and keep room

try adding one one in the middle and then another next to it
image

**Comment by Sylphily** [comment]
Then it double on thickness and looks more wonky

**Comment by Sylphily** [comment]
Guess I can axe that extra seat

**Comment by Sylphily** [comment]
i have no clue why its failing it seems to work and all the pipes are lined up

**Review by Fikou — APPROVED** [review]

**Comment by Apogee-dev** [comment]
Config file needs prefix and namelists fields. You also don't have any firelocks on the bridge (though honestly, I'd suggest placing firelocks on all of the dividing windows, considering how easy they are to break)

File: _maps/configs/scav.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+	"map_name": "Scav-class Drifter",
+	"map_short_name": "Scav-class",
```
Comment (Apogee-dev) [inline _maps/configs/scav.json:3]: "prefix": ISV,
"namelists": insert your namelists of choice here, i'd suggest GENERAL and SPACE at a minimum

**Review by Apogee-dev — CHANGES_REQUESTED** [review]

**Comment by Sylphily** [comment]
and i had just closed DMM pain

**Comment by Apogee-dev** [comment]
You also need to replace all your space tiles with terrain passthrough tiles (and the appropriate area). As a side note, unless they're placed over a floor tile with a ship area, the ship's exterior cameras will simply get deleted.

**Comment by Sylphily** [comment]
camreas have been removed because they didn't seem to work any ways

**Comment by Sylphily** [comment]
also where the hell is the file containing ship names, it dosen't seem to exist?

**Comment by Apogee-dev** [comment]
> also where the hell is the file containing ship names, it dosen't seem to exist?

strings/ship_names.json

**Comment by Sylphily** [comment]
ah i was on the wrong branch, couldn't find them there lol

**Review by Apogee-dev — APPROVED** [review]

**Comment by TerraTheUnwise** [comment]
I very much enjoy this ship. I played a round with it. And looting ruins was very quick and efficient.

**Comment by Latentish** [comment]
I feel like this ship and the Masinyane serve very similar functions. What makes this one special?

**Comment by Sylphily** [comment]
> I feel like this ship and the Masinyane serve very similar functions. What makes this one special?

well it is more scavenger focused, and is just more thematic to scavenging, it has room to RP and room for a bit of medical if you know what you are doing, it also has cargo space for what ever you want, and is quite open, and comfy

**Pull request merged**